projects/netgrif-components-core/src/lib/resources/interface/place-import.ts

```diff
@@ -0,0 +1,10 @@
+import {Position} from './position';
+
+export interface PlaceImport {
+    importId: string;
+    isStatic: boolean;
+    position: Position;
+    stringId: string;
+    title: object;
```

Comment:
why an `object`? Is it not a `string`?

Reply:
its I18NString, but i think we dont have an interface for it, and sometimes its only string and not i18nstring